markdown/clientaccess/client_auth.html.md.erb

```diff
@@ -99,27 +99,25 @@ This example shows how to edit the `pg_hba.conf` file of the master to allow rem
 
 HAWQ allocates some resources on a per-connection basis, so setting the maximum number of connections allowed is recommended.
 
-To limit the number of active concurrent sessions to your HAWQ system, you can configure the `max_connections` server configuration parameter on master or the `seg_max_connections` server configuration parameter on segments. These parameters are *local* parameters, meaning that you must set them in the `hawq-site.xml` file of all HAWQ instances.
+To limit the number of active concurrent sessions to your HAWQ system, you can configure the `max_connections` server configuration parameter. This parameter is a *local* parameter, meaning that you must set them in the `hawq-site.xml` file of all HAWQ instances.
```

Comment:
The previous one seems better:
To limit the number of active concurrent sessions to your HAWQ system, you can configure the `max_connections` server configuration parameter on master or the `seg_max_connections` server configuration parameter on segments.

Except that I think we don't need to mention it's a *local* parameter. We don't need to set it in 'hawq-site.xml' unless we want one specify node to be different than others.